service/src/main/java/bio/terra/workspace/common/logging/model/ActivityLogChangedTarget.java

```diff
@@ -31,7 +31,10 @@ public enum ActivityLogChangedTarget {
   USER,
   WORKSPACE,
   POLICIES,
-  CLOUD_CONTEXT;
+  CLOUD_CONTEXT,
+  GCP_CLOUD_CONTEXT,
```

Comment:
can you log them in the activity log hook?

Reply:
I made https://broadworkbench.atlassian.net/browse/PF-2764 to track restoring the activity log detail.
I wrote two options in there. I'd like your input.